crates/client/sync/src/utils/utility.rs

```diff
@@ -125,3 +125,24 @@ pub fn trim_hash(hash: &Felt) -> String {
 
     format!("{}...{}", prefix, suffix)
 }
+
+pub fn get_directory_size(path: &Path) -> u64 {
```

Comment:
I would prefer if we were using https://docs.rs/rocksdb/latest/rocksdb/struct.DBCommon.html#method.get_column_family_metadata_cf
which returns, for a given column, its size as
```rs
pub struct ColumnFamilyMetaData {
    // The size of this column family in bytes, which is equal to the sum of
    // the file size of its "levels".
    pub size: u64,
    // The name of the column family.
    pub name: String,
    // The number of files in this column family.
    pub file_count: usize,
}
```
Plus, we can do a `sum` gauge and a individual gauge for each column